docker-compose.yml

```diff
@@ -24,8 +24,10 @@ services:
     volumes:
       - .:/var/www/html:cached
       - .docker/php/php-ini-overrides.ini:/usr/local/etc/php/conf.d/99-overrides.ini
+      - .docker/php/001-ssl.conf:/etc/apache2/sites-enabled/001-ssl.conf
```

Comment:
Not needed, since the file is included in the image anyway. Only needed if it would not be included.